skbio/diversity/tests/test_driver.py

```diff
@@ -529,6 +530,48 @@ def test_unweighted_unifrac(self):
                 npt.assert_almost_equal(dm1[id1, id2],
                                         expected_dm[id1, id2], 6)
 
+    def test_unweighted_unifrac_partial(self):
```

Comment:
Can you move all `partial_beta_diversity` test code into a new class `TestPartialBetaDiversity`?